### ISSUE TITLE
BUG: Creating a MySQL database backup / export to a directory with a space fails on Linux Mint

### ISSUE THREAD

**Issue by YourRightWebsite** [beekeeper-studio/beekeeper-studio#2316]
**Describe the bug**
When making a MySQL database backup to a target directory that has a space in it on Linux Mint, the database is never properly selected and the MySQL Dump command that is generated is incorrect, thus resulting in an error when exporting.  Manually adding quotation marks around the output directory in the generated MySQL Dump command and running that on the command line solves the issue.

**To Reproduce**
1. While connected to a MySQL database, go to "Tools" > "Create a Database Backup"
2. Select one or more tables and click "Configure Backup"
3. Under "Output", select a directory that has a space in the name
4. Click "Review"
5. Click "Execute Backup"

**Expected behavior**
A MySQL database backup should be generated to the target directory without errors.

**Version Information (please complete the following information):**
 - OS: Linux Mint 21.3
 - App Version 4.6.4
 - Database type and version: MySQL 5.7.29

**Additional context**
Manually adding quotes around the output directory in the generated mysqldump command allows the backup to run from the command line properly.

**Comment by not-night-but** [comment]
Yeah this is definitely an issue, and one that I've seen before. I know we have code somewhere to escape the spaces properly...

I'm off work right now, should get back at some point tomorrow so I'll take a look when I get a chance.

Thanks for the report!!
